lib/vnet/escalate_daemon_darwin.go

```diff
@@ -35,5 +35,5 @@ func execAdminProcess(ctx context.Context, config daemon.Config) error {
 
 // DaemonSubcommand runs the VNet daemon process.
 func DaemonSubcommand(ctx context.Context) error {
-	return trace.Wrap(daemon.Start(ctx, RunAdminProcess))
+	return trace.Wrap(daemon.Start(ctx, RunDarwinAdminProcess))
```

Comment:
nit: Could RunDarwinAdminProcess be private?

Reply:
no, it is called from tool/tsh/common/vnet_darwin.go, there is a `tsh vnet-admin-setup` command that is used when tsh is built without the `vnetdaemon` tag